src/rust/generators/sysmon-subgraph-generator/src/models/process/create.rs

```diff
@@ -82,7 +77,6 @@ pub fn generate_process_create_subgraph(
     );
 
     graph.add_edge("children", parent.clone_node_key(), child.clone_node_key());
-    graph.add_edge("parent", child.clone_node_key(), parent.clone_node_key());
```

Comment:
@colin-grapl is there a reason this was deleted?

Reply:
It's redundant - there's no need to specify an edge and a reverse edge, one implies the other, and that's handled downstream.